src/targets/neutronrc_900.ini

```diff
@@ -0,0 +1,33 @@
+
+# ********************************
+# Transmitter targets
+# ********************************
+
+
+# ********************************
+# Receiver targets
+# ********************************
+
+[env:NeutronRC_900_RX_via_UART]
+extends = env_common_esp82xx
+upload_speed = 420000
```

Comment:
Why these are defined? There is default values.

Reply:
Removed and fixed other targets